src/main/kotlin/io/appwrite/Query.kt

```diff
@@ -20,11 +20,11 @@ class Query {
 
     fun isNotNull(attribute: String) = "isNotNull(\"${attribute}\")"
 
-    fun between(attribute: String, start: Int, end: Int) = Query.addQuery(attribute, "between", listOf(start, end))
+    fun between(attribute: String, start: Int, end: Int) = "between(\"${attribute}\", ${start}, ${end})"
 
-    fun between(attribute: String, start: Double, end: Double) = Query.addQuery(attribute, "between", listOf(start, end))
+    fun between(attribute: String, start: Double, end: Double) = "between(\"${attribute}\", ${start}, ${end})"
 
-    fun between(attribute: String, start: String, end: String) = Query.addQuery(attribute, "between", listOf(start, end))
+    fun between(attribute: String, start: String, end: String) = "between(\"${attribute}\", \"${start}\", \"${end}\")"
```

Comment:
We should probably `parseValues(start)` and `parseValues(end)`

Reply:
Not strictly necessary since we know exactly what type we're getting here but will address for next release :+1: